pkgs/servers/monitoring/matrix-alertmanager/update.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, writeShellScript
+, coreutils, jq, common-updater-scripts
+, curl, wget, gnugrep, yarn, prefetch-yarn-deps
+}:
+
+writeShellScript "update-matrix-alertmanager" ''
```

Comment:
Why are we not using a shell script with a nix-shell shebang?

Reply:
Why _would_ we use a shell script with nix-shell shebang?

Reply:
Because shell scripts are outdate, and not working with nix flakes that well. It would break for me, as I don't have a NIX_PATH